code/core/src/manager/App.tsx

```diff
@@ -27,6 +27,29 @@ export const App = ({ managerLayoutState, setManagerLayoutState, pages, hasTab }
     document.body.setAttribute('data-shortcuts-enabled', enableShortcuts ? 'true' : 'false');
   }, [enableShortcuts]);
 
+  useEffect(() => {
+    const rootElement = document.getElementById('root');
+    if (!rootElement) {
+      return;
+    }
+
+    const observer = new MutationObserver((mutations) => {
+      mutations.forEach((mutation) => {
+        if (mutation.type === 'attributes' && mutation.attributeName === 'inert') {
+          const hasInert = rootElement.hasAttribute('inert');
+          addons.getChannel().emit('managerFocusTrapChange', hasInert);
```

Comment:
This should use the constant, not a magic string.

Also, the name and value of this constant does not reflect the actual event. I think it should be something like `INERT_ATTRIBUTE_CHANGED`.